rskj-core/src/main/java/co/rsk/rpc/netty/Web3WebSocketServer.java

```diff
@@ -71,8 +75,9 @@ protected void initChannel(SocketChannel ch) throws Exception {
                     ChannelPipeline p = ch.pipeline();
                     p.addLast(new HttpServerCodec());
                     p.addLast(new HttpObjectAggregator(1024 * 1024 * 5));
```

Comment:
I think that maybe we can use this opportunity to move the `1024 * 1024 * 5` thingy to a constant.

Reply:
nice one!